CMakeLists.txt

```diff
@@ -108,28 +108,11 @@ endif()
 
 
 if(APPLE)
-    # For Intel Mac's
-    if(EXISTS /usr/local/opt/qt5)
-        list(APPEND CMAKE_PREFIX_PATH "/usr/local/opt/qt5")
-    endif()
-
-    # For Apple Silicon Mac's
-    if(EXISTS /opt/homebrew/opt/qt5)
-        list(APPEND CMAKE_PREFIX_PATH "/opt/homebrew/opt/qt5")
-    endif()
-    if(EXISTS /opt/homebrew/opt/sqlitefts5)
-        list(PREPEND CMAKE_PREFIX_PATH "/opt/homebrew/opt/sqlitefts5")
-    endif()
-
-    # For Apple Silicon Mac's and install dependencies via our Homebrew tap(sqlitebrowser/homebrew-tap)
-    if(customTap AND EXISTS /opt/homebrew/opt/)
-        list(PREPEND CMAKE_PREFIX_PATH "/opt/homebrew/opt/db4subqt@5")
-        list(PREPEND CMAKE_PREFIX_PATH "/opt/homebrew/opt/db4subsqlitefts@5")
-
-        if(sqlcipher)
-            list(APPEND SQLCIPHER_INCLUDE_DIR "/opt/homebrew/include")
-            list(APPEND SQLCIPHER_LIBRARY "/opt/homebrew/opt/db4subsqlcipher/lib/libsqlcipher.0.dylib")
-        endif()
+    find_package(Homebrew REQUIRED)
+    if (customTap AND sqlcipher)
+        list(PREPEND CMAKE_PREFIX_PATH ${Qt5_DIR} ${SQLCipher_INCLUDE_DIR} ${SQLCipher_INCLUDE_DIR} ${SQLite_DIR})
```

Comment:
`${SQLCipher_INCLUDE_DIR}` seems to be included twice here.  That doesn't *seem* correct to me, but I could be wrong. :smile:

Reply:
Oh no, that's a mistake. Thanks for letting me know :). Resolve via [`29838cb` (#3472)](https://github.com/sqlitebrowser/sqlitebrowser/pull/3472/commits/29838cbacb6d98e50dad9ab4234397561e906577)